bin/executor/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+name = "node-executor"
```

Comment:
Would it be possible to use the Parity crate `node-executor` which is the member of the `substrate` monorepo?

Reply:
Nope. On L31 of  `bin/executor/src/lib.rs`, I had to inject our WASM host functions to the executor.